pyirf/statistics.py

```diff
@@ -32,8 +32,11 @@ def li_ma_significance(n_on, n_off, alpha=0.2):
 
     scalar = is_scalar(n_on)
 
-    n_on = np.array(n_on, copy=False, ndmin=1)
-    n_off = np.array(n_off, copy=False, ndmin=1)
+    # Cast everything into float64 to avoid numeric instabilties
+    # when multiplying very small and very big numbers to get t1 and t2
+    n_on = np.array(n_on, copy=False, ndmin=1, dtype=np.float64)
```

Comment:
asanyarray preserves array subtypes I think....

Reply:
It does but does not support the `ndmin` argument needed in the current form of this function (specifically the boolean mask applied to {t1, t2}). Having both would ofc. be quite easy but I do not really see the usecase for theoretically preserving the subtype here if we forcefully overwrite it to f64. But maybe I do not completely get your point here :sweat_smile:

Reply:
"keeping subtype" means that e.g. if you insert a pd.Series or a `u.Quantity` the result will still be that, not the base `np.ndarray`